oddeven.py

```diff
@@ -1,5 +1,6 @@
+#Oddeven program using selection process: 
 n=4
```

Comment:
write a program to get the `n` value from terminal

https://stackoverflow.com/questions/5404068/how-to-read-keyboard-input